code/src/components/ErrorBoundary.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Error from "./Error";
+
+class ErrorBoundary extends React.Component {
```

Comment:
Is this the "old" way of writing code in React? Curious to know why you found this easier to use than the modern way.

Reply:
I have no idea - I was following the guide for writing Error boundaries on the React documentation page.